src/components/button/button.jsx

```diff
@@ -25,6 +25,7 @@ export default function Button(props) {
 Button.propTypes = {
   className: PropTypes.string,
   color: PropTypes.oneOf([
+    'blank',
```

Comment:
Looks like we need to get rid of `secondary` below...friendly reminder as I'm assuming you're planning to do that.

Reply:
I decided against it. It's part of a commit @ https://github.com/mavenlink/design-system/pull/24/commits/de3f2daf627a048d277d0f635a760d7d468027b9 Essentially, it helps to extract the appropriate class names.